server/src/instant/scripts/welcome_email.clj

```diff
@@ -0,0 +1,87 @@
+(ns instant.scripts.welcome-email
+  "Send welcome emails to our users"
+  (:require
+   [instant.util.date :as date]
+   [instant.postmark :as postmark]
+   [chime.core :as chime-core]
+   [clojure.tools.logging :as log]
+   [instant.grab :as grab]
+   [instant.jdbc.aurora :as aurora]
+   [instant.jdbc.sql :as sql]
+   [instant.flags :as flags])
+  (:import
+   (java.time Period DayOfWeek)))
+
+(defn find-new-users
+  ([] (find-new-users (aurora/conn-pool :read)))
+  ([conn]
+   (sql/select
+    :welcome-email-users
+    conn
+    ["SELECT email
```

Comment:
Maybe we could add a `sent_welcome_email` flag to the user so that we don't accidentally send the same email twice?